sdk/storage/azure-storage-file-datalake/src/test/java/com/azure/storage/file/datalake/UrlTests.groovy

```diff
@@ -0,0 +1,98 @@
+package com.azure.storage.file.datalake
+
+class UrlTests extends APISpec {
+
+    def "test custom url"() {
+        when:
+        def customUrl = "https://www.customstorageurl.com"
```

Comment:
What is the expected return for `accountName` in this case?

Reply:
Custom URLs dont actually work for Datalake, so the user should get some exception probably a 404